Packs/CVE_2022_30190/Playbooks/playbook-CVE-2022-30190_-_MSDT_RCE_README.md

```diff
@@ -0,0 +1,63 @@
+On **May 27th**, a new Microsoft Office Zero-Day was discovered by Nao_sec. 
+
+The new Zero-Day is a remote code execution vulnerability that exists when MSDT is called using the URL protocol from a calling application such as Word. 
+
+On **May 30th**, Microsoft assigned **CVE-2022-30190** to the MSDT vulnerability, aka **Follina vulnerability**.
+
+The playbook includes the following tasks:
```

Comment:
This playbook includes the following tasks:
* Collect detection rules.
* Exploitation patterns hunting using Cortex XDR - XQL Engine and 3rd party SIEM products.
* Cortex XDR BIOCs coverage.
* Provides Microsoft workarounds and detection capabilities.

Reply:
Done